cfgov/jinja2/v1/_includes/templates/content-sidebar.html

```diff
@@ -0,0 +1,19 @@
+{% for block in page.sidefoot %}
```

Comment:
@kave sorry - meant to respond to this earlier. I can add it back in, but I thought the context is included?  If not, shouldn't something be breaking?